frontend/src/utilities/useEnableApplication.tsx

```diff
@@ -56,7 +56,7 @@ export const useEnableApplication = (
 
   React.useEffect(() => {
     let cancelled = false;
-    let watchHandle: ReturnType<typeof setTimeout>;
+    let watchHandle: ReturnType<typeof setTimeout> | undefined;
```

Comment:
Remove the if condition in the return statement.
```suggestion
    let watchHandle: ReturnType<typeof setTimeout>;
```